lib/newman_conway.rb

```diff
@@ -1,7 +1,30 @@
 
 
-# Time complexity: ?
-# Space Complexity: ?
+# Time complexity: O(n)
+# Space Complexity: O(1)
 def newman_conway(num)
```

Comment:
How would you have O(1) space complexity?  You're building an array.  Otherwise well done.